graphwalker-io/src/test/java/org/graphwalker/io/factory/json/JsonContextFactoryTest.java

```diff
@@ -235,4 +235,10 @@ public void PetClinicWithSimpleMachine() throws IOException {
       logger.debug(e.getName());
     }
   }
+  
+  @Test
+  public void acceptDependencyJsonTest() {
```

Comment:
@KristianKarl it was not finished yet :) However, thanks for the directions. I plan to use the DependencyModel.json for the example page.